lib/matplotlib/patches.py

```diff
@@ -4182,49 +4182,90 @@ def __init__(self, posA=None, posB=None, *,
                  patchA=None, patchB=None, shrinkA=2, shrinkB=2,
                  mutation_scale=1, mutation_aspect=1, **kwargs):
         """
-        There are two ways for defining an arrow:
+        **Defining the arrow position and path**
 
-        - If *posA* and *posB* are given, a path connecting two points is
-          created according to *connectionstyle*. The path will be
-          clipped with *patchA* and *patchB* and further shrunken by
-          *shrinkA* and *shrinkB*. An arrow is drawn along this
-          resulting path using the *arrowstyle* parameter.
+        There are two ways to define the arrow position and path:
 
-        - Alternatively if *path* is provided, an arrow is drawn along this
-          path and *patchA*, *patchB*, *shrinkA*, and *shrinkB* are ignored.
+        - **Start, end and connection**:
+          The typical approach is to define the start and end points of the
+          arrow using *posA* and *posB*. The curve between these two can
+          further be configured using *connectionstyle*.
+
+          If given, the arrow curve is clipped by *patchA* and *patchB*,
+          allowing it to start/end at the border of these patches.
+          Additionally, the arrow curve can be shortened by *shrinkA* and
+          to create a margin between the arrow and the patch it is pointing
+          to.
```

Comment:
Should this be `*shrinkA* and *shrinkB*`?
```suggestion
          Additionally, the arrow curve can be shortened by *shrinkA* and *shrinkB*
          to create a margin between the arrow and the patch it is pointing
          to.
```
Also, the shrink is from the start/end points, is it not? Which may or may not be related to a patch...